test/index.spec.js

```diff
@@ -193,3 +195,43 @@ test.each(cases)('node: %s', (name, input) => {
 
   expect(code).toMatchFile();
 });
+
+test('targets: modern', () => {
+  let {
+    options: { targets },
+  } = transform('', {
+    targets: { browsers: 'defaults', esmodules: true },
+    presets: [[preset, { modules: false }]],
+  });
+
+  let browsers = browserslist('defaults and supports es6-module');
+
+  let result = [
+    '// Babel targets',
+    JSON.stringify(targets, null, 2),
+    '// Browerslist targets',
+    JSON.stringify(browsers, null, 2),
+  ].join('\n');
+
+  expect(result).toMatchFile();
+});
+
+test('targets: legacy', () => {
+  let {
+    options: { targets },
+  } = transform('', {
+    targets: { browsers: 'defaults' },
+    presets: [[preset, { modules: false }]],
+  });
+
+  let browsers = browserslist('defaults');
```

Comment:
```suggestion
  let browsers = browserslist('defaults and not supports es6-module');
```
?

And the list will be empty, right?

Reply:
Quite empty yes :) https://browsersl.ist/#q=defaults+and+not+supports+es6-module

Reply:
we can ignore the legacy bundle and we will clean it after